### PR TITLE
First pass of updates for highlander branch

### PR DESCRIPTION
**Pull request by tlisonbee**

### CODE REVIEW

File: docs/administration-guide/deleting-an-environment.md [hunk 1]
```diff
@@ -3,28 +3,8 @@ layout: documentation
...
-    - So if you have manually added instances that are not associated with the above stacks that you deleted, you will need to manually delete them.
-4. Now that the VPC and all the AWS Resources have been deleted the only thing remaining is the certs we uploaded. Unfortunately you must use the AWS CLI for this step. Run the following commands
-    - `aws iam list-server-certificates | grep ${env}`
-    - For each of the certs that are in the `/cerberus/${env}/` path run the following command to delete it, using the cert name which is the last section of the arn ex: `cms_7e0a04d4-77e7-4c5a-8905-9cbaffd0c291`
-    - `aws iam delete-server-certificate --server-certificate-name cms_7e0a04d4-77e7-4c5a-8905-9cbaffd0c291`
+1. Use the `delete-environment` CLI command which will delete all of the CloudFormation stacks.
+1. Manually delete all KMS keys that were associated with the environment.
+1. Manually delete lambdas associated with the environment.
+1. Manually delete any remaining S3 buckets associated with the environment.
+1. Manually delete any remaining RDS snapshots associated with the environment.
```
Comment (tlisonbee) [inline docs/administration-guide/deleting-an-environment.md]: I guessed on this procedure (I've never done it before).  Please review for accuracy.

**Comment by tlisonbee** [comment]
This PR is easier to review if you checkout the branch locally and run the whole website using the commands from the README.

Also, it is missing all of the `create an environment` steps.

File: docs/administration-guide/deleting-an-environment.md (hunk 1 above)
Comment (sdford) [reply docs/administration-guide/deleting-an-environment.md]: S3 buckets are taken care of in the first step, so long as they are in one of the CF stacks for the environment.

Also, maybe add an example delete command here for step 1?

**Review by sdford — APPROVED** [review]

File: _posts/2018-01-04-next-generation-architecture.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+6. Deploy rate limiting lambda
+7. Verify new environment is healthy
+
+## Cut Over
+
+1. Temporarily stop writing to Cerberus
+2. Trigger backup job for old environment (using CLI version 3.3.1)
+3. Retrieve a token from Cerberus with admin permissions (log into dashboard, or use token bash script)
+4. Export VAULT_TOKEN variable in terminal window where running CLI
+5. Once backup job is finished, restore backup:
```
Comment (sdford) [inline _posts/2018-01-04-next-generation-architecture.md]: Maybe a note here to whitelist your IP before restore. Or a warning that you may get rate limited if you have the rate limiting lambda set up.

**Review by fieldju — APPROVED** [review]

File: docs/architecture/vault.md [hunk 3]
```diff
@@ -1,63 +0,0 @@
...
-title: Vault
----
-
-<a target="_blank" onclick="trackOutboundLink('https://www.vaultproject.io/')" href="https://www.vaultproject.io/">Vault</a> is a tool created by <a target="_blank" onclick="trackOutboundLink('https://www.hashicorp.com/')" href="https://www.hashicorp.com/">Hashicorp</a> for managing 
-secrets. Cerberus uses Vault for storage and securing access to data.
-
-Consul is a tool for service discovery, monitoring and configuration. In Cerberus, Consul is the storage backend for 
-Vault and provides high availability.
-
-# Safe Deposit Box (SDB)
```
Comment (sdford) [inline docs/architecture/vault.md:12]: Might be worth it to keep this somewhere in the docs?

Maybe it's already on another page.

Comment (tlisonbee) [reply docs/architecture/vault.md:12]: I think I moved / rewrote

**Pull request merged**